backend/routes/topTenToppings.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from 'express'
+import { getToppingsCollection } from '../db'
+import expandToppingsCategories from '../expandToppingsCategories'
+import { Topping, ToppingWithCategoryId } from '../topping'
+import { SuccessResponse } from '../response'
+
+export default async function topTenToppings(
+  _: Request, response: Response<SuccessResponse<Topping[]>>)
```

Comment:
It's because I'm not doing anything with the request and the underscore makes my editor stop complaining about that (and some tools complain about that too). And I can't avoid having the request argument because I need to get to the response

Reply:
🤠